### PR TITLE
Makes DisplayNameProvider public.

### PR DESCRIPTION
**Pull request by jas-valgotar**

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/SingleSourceDisplayNameProvider.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 namespace Microsoft.PowerFx.Core
 {
-    internal class SingleSourceDisplayNameProvider : DisplayNameProvider
+    public class SingleSourceDisplayNameProvider : DisplayNameProvider
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/SingleSourceDisplayNameProvider.cs:14]: nit: can we keep the class private, and instead have a static builder on the base DisplayNameProvider class?

Comment (jas-valgotar) [reply src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/SingleSourceDisplayNameProvider.cs]: good point, updated.

**Review by MikeStall — APPROVED** [review]

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**